server/configuration.go

```diff
@@ -80,6 +81,14 @@ func (p *Plugin) setConfiguration(configuration *configuration) {
 func (p *Plugin) OnConfigurationChange() error {
 	var configuration = new(configuration)
 
+	if p.GetPluginAPI().GetConfig().ConnectedWorkspacesSettings.EnableRemoteClusterService != nil && *p.GetPluginAPI().GetConfig().ConnectedWorkspacesSettings.EnableRemoteClusterService {
+		return fmt.Errorf("remote Cluster Service it's required and is currently not enabled")
```

Comment:
Remote Cluster service is automatically enabled in main product if Shared Channels service is enabled. I think the check below is enough.

Reply:
I've done a quick search and didn't see where the code does this, even [the tests](https://github.com/mattermost/mattermost/blob/ae35dd93a98f236e914ef4bbfc97e6dfe6ff8d58/server/channels/api4/shared_channel_test.go#L25) set the values separately and there's no test handling that case. I'd rather let them both since those are different features and we need them both.

Reply:
Sorry - wrong line of code:  https://github.com/mattermost/mattermost/blob/1c46cbe646c1187b697edec3bd3dba0e31f0b9fc/server/channels/app/server.go#L564

Reply:
That line seems to be checking the same both settings I'm checking in this PR.